sdk/cosmos/azure-spring-data-cosmos/README.md

```diff
@@ -293,7 +293,7 @@ public interface UserRepository extends CosmosRepository<User, String> {
 #### QueryAnnotation : Using annotated queries in repositories
 Azure spring data cosmos supports specifying annotated queries in the repositories using `@Query`.
 - Examples for annotated queries in synchronous CosmosRepository:
-<!-- embedme src/samples/java/com/azure/spring/data/cosmos/AnnotatedQueriesUserRepositoryCodeSnippet.java#L11-L17 -->
+<!-- embedme src/samples/java/com/azure/spring/data/cosmos/AnnotatedQueriesUserRepositoryCodeSnippet.java#L11-L20 -->
```

Comment:
Same as this one, AnnotatedQueriesUserReactiveRepositoryCodeSnippet also needs to be updated in the readme.